package.json

```diff
@@ -15,7 +15,8 @@
     "test:ios": "cd native/iosTest; xcodebuild -workspace WatermelonTester.xcworkspace -scheme 'WatermelonTester' -destination 'name=iPhone 8' test",
     "swiftlint": "cd native; swiftlint",
     "ktlint": "cd native/androidTest; ./gradlew ktlint",
-    "ktlint:format": "cd native/androidTest; ./gradlew ktlintFormat"
+    "ktlint:format": "cd native/androidTest; ./gradlew ktlintFormat",
+    "android:emulator": "export JAVA_OPTS=\"-XX:+IgnoreUnrecognizedVMOptions --add-modules java.se.ee\"; node ./scripts/emulator.js"
```

Comment:
This breaks it for me:

```
 ~/Documents/Nozbe/WatermelonDB   android-emu-script ⇕ yarn android:emulator
yarn run v1.9.4
$ export JAVA_OPTS="-XX:+IgnoreUnrecognizedVMOptions --add-modules java.se.ee"; node ./scripts/emulator.js
Error: no se ha encontrado o cargado la clase principal java.se.ee
child_process.js:640
    throw err;
    ^

Error: Command failed: $ANDROID_HOME/tools/bin/sdkmanager --list | grep "system-images/"
Error: no se ha encontrado o cargado la clase principal java.se.ee

    at checkExecSyncError (child_process.js:600:11)
    at execSync (child_process.js:637:13)
    at Object.<anonymous> (/Users/radex/Documents/Nozbe/WatermelonDB/scripts/emulator.js:13:14)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:266:19)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
 ✘  ~/Documents/Nozbe/WatermelonDB   android-emu-script ⇕ java -version
java version "1.8.0_131"
Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)
 ~/Documents/Nozbe/WatermelonDB   android-emu-script ⇕ javac -version
javac 1.8.0_131
 ~/Documents/Nozbe/WatermelonDB   android-emu-script ⇕

```

Reply:
Huh, can you try running simply `node ./scripts/emulator.js` to see if this works?

Reply:
It does work! Both opening existing and creating a new emulator works